inc/admin/admin.php

```diff
@@ -497,6 +495,8 @@ function rocket_analytics_data() {
 	$data['active_plugins']    = rocket_get_active_plugins();
 	$data['locale']            = $locale[0];
 	$data['multisite']         = is_multisite();
+	$data['cdn_cnames']        = count( $data['cdn_cnames'] );
+	$data['sitemaps']          = array_map( 'rocket_clean_exclude_file', $data['sitemaps'] );
```

Comment:
If `$data['cdn_cnames']` or `$data['sitemaps']` isn't set or isn't an array, you'll get a notice.

Reply:
In the callback function of register_setting, those values are always set: if not set or empty from the user input, it defaults to an empty array.

Do you think it still needs a check?

Reply:
OK it forces a fully filled array on save, but it doesn't prevent `get_option()` from outputting `false` if the option has been deleted by an evil dude 😁 
And in that case, those keys aren't set of course.

Reply:
OK, your last commit fixes everything :)